SU-HSDO/suhumsci
# STN-219: QA Feedback from Sprint 13

# READY FOR REVIEW

## Summary
- Adjusts the `.hb-pill` styles when nested `divs` are used.
- Deletes the `.hb-subheading` utility.
- Removes default category styles from `.hb-card` and fixes the `hb-categories` utility when used on an `.entity-reference` field.
- Adds padding to `.hb-well`
- Fixes the mobile table headers for the table pattern. All rows should now have table headers.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Confirm that the categories in horizontal card are unstyled and will then work when adding the `hb-categories` class.
1.  Add the `hb-categories` class to the research areas field on a person page and confirm the double border is gone.
1.  Add `hb-well` to any field on the person page and ensure that there is both padding and a background color.
1. Add the following `hb-pill-list` and `hb-pill-link-list` to a field that has multiple links, such as research areas, or info links.
1. View the courses page on mobile and check that all rows have their headings. Scroll all the way through the content, we didn't realize that this table was also broken but the breakage happened further down the page.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
